index.html

```diff
@@ -2324,6 +2324,52 @@ <h3>Presentations</h3>
 [[?VC-JOSE-COSE]] specification.
         </p>
 
+        <section>
+          <h4>Enveloped Verifiable Credentials</h4>
+
+          <p>
+It is possible to include one or more <a>verifiable credentials</a> in a
+<a>verifiable presentation</a> that have been secured using a securing mechanism
+that "envelopes" the payload, such as [[[?VC-JOSE-COSE]]] [[?VC-JOSE-COSE]].
+This can be accomplished by associating the `verifiableCredential` property with
+an object that has a `type` of `EnvelopedVerifiableCredential`.
+          </p>
+
+          <dl>
+            <dt id="defn-EnvelopedVerifiableCredential">EnvelopedVerifiableCredential</dt>
+            <dd>
+Used to associate an object containing an enveloped <a>verifiable credential</a>
+with the `verifiableCredential` property in a <a>verifiable presentation</a>.
+The `@context` value of the object MUST follow all of the "MUST" statements in
+Section <a href="#contexts"></a>. The `id` value of the object MUST be a `data:`
+URL that expresses a secured <a>verifiable credential</a> using an enveloping
```

Comment:
```suggestion
URL [[RFC2397]] that expresses a secured <a>verifiable credential</a> using an enveloping
```

Just adding a reference to the Data URL...